.gitignore

```diff
@@ -9,3 +9,4 @@ node_modules
 coverage
 npm-debug.log
 *.swp
+.vscode
```

Comment:
this change is unrelated to the PR and stuff like that should be added to your global gitignore file instead

Reply:
Oh, sure